src/rules.rs

```diff
@@ -100,6 +102,7 @@ mod test {
         assert_eq!(rules.min_word_count, 1);
         assert_eq!(rules.max_word_count, 14);
         assert_eq!(rules.min_characters, 0);
+        assert_eq!(rules.max_characters, 999);
```

Comment:
Where is 999 coming from? I think the tests failed before so you didn't get an automated comment here. This should be `usize::MAX`.